### DISCUSSION TITLE
Does RESTEasy 6 supports Jetty 9.4.x?

### DISCUSSION THREAD

**Discussion by selva3k** [resteasy/resteasy#3045] [Q&A]
Jetty version
9.4.35

Java version
1.8

Question
Does RESTEasy 6 supports Jetty 9.4.35(java version:1.8)? If yes, please share some material for the integration (if you have any reference)

**Comment by jamezp** [comment]
It does not. RESTEasy 6 targets Jakarta RESTful Web Services 3.0 which is part of Jakarta EE 9. It requires Servlet 5.0 which uses the `jakarta` namespace. It would only work with Jetty 11. However, the only difference between RESTEasy 5 and RESTEasy 6 is the `jakarta` namespace.

**Reply by selva3k** [reply]
Thanks. Can we use RESTEasy 5 with Jetty 9.4.35(java version:1.8)?    In other words, which latest RESTEasy version is compatible with Jetty 9.4.35(java version:1.8)?

**Reply by jamezp** [reply]
While I haven't personally test it, yes. RESTEasy 5 targets Jakarta EE 8 so if it's a compliant servlet container it should be fine.
